### PR TITLE
Used git clone instead to download source from a branch of a tag

### PR DESCRIPTION
**Pull request by yitam**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/832)

### CODE REVIEW

File: buildscripts/buildtools.py [hunk 1]
```diff
@@ -193,48 +193,33 @@ def write_lines_to_copy_source(driver, file):
...
-
-            print('Extracting ' + file + ' ...')
-            zip = zipfile.ZipFile(file)
-            zip.extractall()
-            zip.close()
-            
             msphpsqlFolder = os.path.join(temppath, 'msphpsql-' + branch)
+            
+            url = 'https://github.com/' + repo + '/msphpsql.git'
+            command = 'git clone ' + url + ' -b ' + branch + ' --single-branch --depth 1 ' + msphpsqlFolder
```
Comment (david-puglielli) [inline buildscripts/buildtools.py]: git is always in the PATH, right? Is there a check for this somewhere else?

File: buildscripts/buildtools.py [hunk 2]
```diff
@@ -193,48 +193,33 @@ def write_lines_to_copy_source(driver, file):
         file.write('@CALL ROBOCOPY ' + source + ' ' + dest + ' /s /xx /xo' + os.linesep)
     
     @staticmethod
-    def download_msphpsql_source(repo, branch, dest_folder = 'Source', clean_up = True):
```
Comment (david-puglielli) [inline buildscripts/buildtools.py]: clean_up has been removed entirely and it's not used anywhere else?

Comment (yitam) [reply buildscripts/buildtools.py]: yes, it was a default parameter, which was never used

File: buildscripts/buildtools.py (hunk 1 above)
Comment (yitam) [reply buildscripts/buildtools.py]: I'm not sure what you meant?

Comment (david-puglielli) [reply buildscripts/buildtools.py]: Is there a check somewhere else in the script that git is actually installed and in the PATH?

Comment (yitam) [reply buildscripts/buildtools.py]: This is meant to be a sample script, which assumes Git is in the path. FYI please check README .

**Pull request merged**